Firebase/Auth/Source/Backend/RPC/FIRVerifyClientRequest.m

```diff
@@ -55,7 +55,7 @@ - (nullable id)unencodedHTTPRequestBodyWithError:(NSError *__autoreleasing _Null
   if (_isSandbox) {
     postBody[kIsSandboxKey] = @YES;
   }
-  return postBody;
+  return [postBody copy];
```

Comment:
Why is the copy added here and on several other returns?

Reply:
`copy` makes the returned string immutable and copying in the future will be free of time and space.